docs/examples/plot_spectral_harmonics.py

```diff
@@ -0,0 +1,166 @@
+# coding: utf-8
+"""
+==================
+Spectral harmonics
```

Comment:
"spectral harmonics": 8k hits on Google
"harmonic spectrum": 192k hits on Google

Reply:
Yeah, fair enough :joy:

Reply:
Though I would like to avoid too much potential for confusion between this and the harmonic *component* of the spectrogram via hpss.